ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/BlockHeader.java

```diff
@@ -225,6 +230,9 @@ public void writeTo(final RLPOutput out) {
     if (baseFee != null) {
       out.writeUInt256Scalar(baseFee);
     }
+    if (excessDataGas != null) {
```

Comment:
I think the next expected field is the withdrawalsRoot followed by the excessDataGas. So could merge this in after my PR so this is the last field or just skip the next field.

Reply:
yes, merged your PR and moved after `withdrawalsRoot`